sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -662,6 +665,110 @@ public Mono<Map<String, Map<String, String>>> getBulkSecret(GetBulkSecretRequest
+      }
+      if (lockOwner == null || lockOwner.isEmpty()) {
+        throw new IllegalArgumentException("LockOwner cannot be null or empty.");
+      }
+      if (expiryInSeconds < 0) {
+        throw new IllegalArgumentException("ExpiryInSeconds cannot be negative.");
+      }
+
+
+      Map<String, Object> jsonMap = new HashMap<>();
+      jsonMap.put("resourceId", resourceId);
+      jsonMap.put("lockOwner", lockOwner);
+      jsonMap.put("expiryInSeconds", expiryInSeconds);
+
+      byte[] requestBody = INTERNAL_SERIALIZER.serialize(jsonMap);
+
+      String[] pathSegments = new String[]{DaprHttp.ALPHA_1_API_VERSION, "lock", stateStoreName};
+
+      return Mono.subscriberContext().flatMap(
+              context -> this.client
+                      .invokeApi(DaprHttp.HttpMethods.POST.name(), pathSegments, null, requestBody, null, context)
+      ).flatMap(response -> {
+        try {
+          Map m = INTERNAL_SERIALIZER.deserialize(response.getBody(), Map.class);
+          if (m == null) {
+            return Mono.just(Boolean.FALSE);
+          }
+          return Mono.just((Boolean) m.get("success"));
+        } catch (Exception ex) {
+          return DaprException.wrapMono(ex);
+        }
+      });
+    } catch (Exception ex) {
+      return DaprException.wrapMono(ex);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Mono<UnlockResponseStatus> unlock(UnlockRequest request) {
+    try {
+      final String stateStoreName = request.getStoreName();
+      final String resourceId = request.getResourceId();
+      final String lockOwner = request.getLockOwner();
+
+      if ((stateStoreName == null) || (stateStoreName.trim().isEmpty())) {
+        throw new IllegalArgumentException("State store name cannot be null or empty.");
+      }
+      if (resourceId == null || resourceId.isEmpty()) {
+        throw new IllegalArgumentException("ResourceId cannot be null or empty.");
+      }
+      if (lockOwner == null || lockOwner.isEmpty()) {
+        throw new IllegalArgumentException("LockOwner cannot be null or empty.");
+      }
+
+
+      Map<String, Object> jsonMap = new HashMap<>();
+      jsonMap.put("resourceId", resourceId);
+      jsonMap.put("lockOwner", lockOwner);
+
+      byte[] requestBody = INTERNAL_SERIALIZER.serialize(jsonMap);
+
+      String[] pathSegments = new String[]{DaprHttp.ALPHA_1_API_VERSION, "unlock", stateStoreName};
+
+      return Mono.subscriberContext().flatMap(
+              context -> this.client
+                      .invokeApi(DaprHttp.HttpMethods.POST.name(), pathSegments, null, requestBody, null, context)
+      ).flatMap(response -> {
+        try {
+          Map m = INTERNAL_SERIALIZER.deserialize(response.getBody(), Map.class);
+          if (m == null) {
+            return Mono.just(UnlockResponseStatus.INTERNAL_ERROR);
+          }
+
+          Integer statusCode = (Integer) m.get("status");
```

Comment:
Same here - can we make `"status"` a constant so it's guaranteed to be consistent anywhere it gets used?

Reply:
For the time being, only the lock api is in use, and if there is a scene in the future, it will be extracted